tests/integration/utils.py

```diff
@@ -3,6 +3,7 @@
 from collections import namedtuple
 import time
 
+import urllib
```

Comment:
Since this is part of the standard library, could you please move this to the previous section?
```suggestion
from collections import namedtuple
import time
import urllib

```